tools/ci/scipy-openblas-macos-x86-64.pc

```diff
@@ -0,0 +1,12 @@
+libdir=/Users/runner/work/numpy/numpy/host-env/scipy_openblas64/lib
```

Comment:
This file should be generated by a script. The problem with `scipy-openblas` is that it has a function to generate the `.pc` file, but it relies on `dlopen`-ing the shared library, which won't work for cross builds.